pymc_extras/inference/pathfinder/pathfinder.py

```diff
@@ -1685,17 +1688,24 @@ def fit_pathfinder(
         Additional keyword arguments for the Pathfinder algorithm.
     compile_kwargs
         Additional keyword arguments for the PyTensor compiler. If not provided, the default linker is "cvm_nogc".
+    initvals: dict | None = None
+        Initial values for the model parameters, as str:ndarray key-value pairs. Paritial initialization is permitted.
+        If None, the model's default initial values are used.
 
     Returns
     -------
-    arviz.InferenceData
+    :class:`~arviz.InferenceData`
         The inference data containing the results of the Pathfinder algorithm.
 
     References
     ----------
     Zhang, L., Carpenter, B., Gelman, A., & Vehtari, A. (2022). Pathfinder: Parallel quasi-Newton variational inference. Journal of Machine Learning Research, 23(306), 1-49.
     """
 
+    if initvals is not None:
```

Comment:
Shouldn't mutate the model, make a copy perhaps if there's no better way to just forward the initvals

Reply:
I would normally agree. However, I tried it, but model.copy() does not produce a working model sometimes - most notably when any transformations are used. 

Should I use some other copy function?

Reply:
@velochy you want the `clone_model` function https://github.com/pymc-devs/pymc/blob/2842401f95de74ab37b7750cff455af28cddaffa/pymc/model/fgraph.py#L374

Reply:
Thank you for that @zaxtax 
Updated the code. Also added a test to check if initvals are used correctly.